front/lib/swr.ts

```diff
@@ -1461,6 +1464,37 @@ export function useVaultDataSourceViews({
   };
 }
 
+export function useVaultDataSourceViewsWithConnector({
+  category,
+  disabled,
+  vaultId,
+  workspaceId,
+}: {
```

Comment:
`nit`
Extract the props outside.

Reply:
in a dedicated type ? we don't have any of this in swr.ts ?

Reply:
Ah yeah missed swr file.